docs/extras/simp.md

```diff
@@ -0,0 +1,60 @@
+# Simp #
+
+The `simp` tactic works by applying "a conditional term rewriting system" (I don't really know what this means and would be interested to know more), to try and simplify the goal. You can actually watch what it's doing (which to the untrained eye looks like trying to apply lots of lemmas tagged with simp until it either makes progress or stops) by writing `set_option trace.simplify true` in your code. If you do this in the below
+
+```lean
+namespace xena
+
+definition cong (a b m : ℤ) : Prop := ∃ n : ℤ, b - a = m * n
+
+notation a ` ≡ ` b ` mod ` m  := cong a b m 
+set_option trace.simplify true
+theorem cong_refl (m : ℤ) : ∀ a : ℤ, a ≡ a mod m :=
+begin
+intro a,
+unfold cong,
+existsi (0:ℤ),
+simp
+end
+
+end xena
+```
+
+you will discover that Lean seems to apply simp when you do an unfold as well (at least I am assuming that this is what's going on -- unfold is also underlined in green).
```

Comment:
That is what is going on - `unfold` is defined using the `simp` framework. I don't think there is much difference between `unfold foo bar` and `simp only [foo, bar]`.